CHANGELOG.rst

```diff
@@ -40,14 +39,15 @@ CHANGELOG
   ``x-systemd.automount x-systemd.requires=lnet.service`` for systemd based systems.
 * Increase the number of hosts that can be processed by scaling daemons in a single batch from 50 to 200. This
   improves the scaling time especially with increased ASG launch rates.
-* Change default sshd config in order to disable X11 forwarding and allow only secure ciphers.
+* Change default sshd config in order to disable X11 forwarding and update the list of supported ciphers
   significantly increases scaling speed when ASG launch rate is raised.
 * Increase faulty node termination timeout from 1 minute to 5 in order to give some additional time to the scheduler
   to recover when under heavy load.
-* Add ``--cluster-template``, ``--vpc-id``, ``-no-public-ip`` and ``--subnet-id`` parameters to ``pcluster createami`` command
+* Extend ``pcluster createami`` to include ``--cluster-template``, ``--vpc-id``, ``-no-public-ip`` and ``--subnet-id``
```

Comment:
Extended ``pcluster createami`` command to specify the VPC and network settings when building the AMI.